components/tidb_query_executors/src/runner.rs

```diff
@@ -510,6 +508,17 @@ impl<SS: 'static> BatchExecutorsRunner<SS> {
         self.out_most_executor.collect_storage_stats(dest);
     }
 
+    pub fn collect_kv_read_time(&mut self, dest: &mut ExecSummary) {
+        if !self.collect_exec_summary {
+            return;
+        }
+        let result = &self.exec_stats.summary_per_executor;
+        if let Some(exec_stat) = result.first() {
+            dest.clone_from(exec_stat);
+        }
+        self.exec_stats.clear();
```

Comment:
```suggestion
```

Reply:
If remove this line, where to clear the `exec_stats`😳